paper/content/chapter2/chapter2_1.tex

```diff
@@ -1,9 +1,156 @@
+
+Mit der Festlegung von Werten je Figur werden vier verschiedene Ziele erreicht:
+
+\begin{enumerate}
+    \item Vermeidung des Austauschs einer kleinen Figur gegen drei Bauern
+    \item Dem Computer signalisieren, dass das halten des Läuferpaars vorteilhaft ist
+    \item Vermeidung des Austauschs von zwei kleinen Figuren gegen einen Turm und einen Bauern
+    \item Verbleiben bei der menschlichen Schacherfahrung
+\end{enumerate}
+
+\noindent Der erste Punkt wird durch Gleichung~\ref{eq:piece-values_first-result} erfüllt.
+
+\begin{equation} \label{eq:piece-values_first-result}
+\begin{split}
+    L > 3B \\
+    S > 3B
+\end{split}
+\end{equation}
+
+\noindent Zwar gibt es durchaus Positionen, in denen drei Bauern wertvoller als eine kleine Figur sind, jedoch ist es im Allgemeinen besser eine kleine Figur zu behalten, da im Spielverlauf die Bauern individuell attackiert werden können und deren Wert als Dreier-Figuren-Gespann verloren geht [\cite{Wiki2018}].
+Der zweite genannte Punkt wird durch Gleichung~\ref{eq:piece-values_second-result} erreicht.
+
+\begin{equation} \label{eq:piece-values_second-result}
+    L > S
+\end{equation}
+
+\noindent Zwar garantiert diese Gleichung kein Halten des Läuferpaars, da am Ende ein Läufer gegen einen Springer stehen kann, dennoch ist es eine Tatsache, dass Spieler oftmals Springer mit Läufern schlagen und nicht Läufer mit Springern [\cite{Wiki2018}].
+Die ersten beiden Gleichungen~\ref{eq:piece-values_first-result} und~\ref{eq:piece-values_second-result} führen zusammen zu Gleichung~\ref{eq:piece-values_first-and-second-result-combined}.
+
+\begin{equation} \label{eq:piece-values_first-and-second-result-combined}
+    L > S > 3B
+\end{equation}
+
+\noindent Der dritte Punkt wird zwar durch Gleichung~\ref{eq:piece-values_third-result} erreicht, dennoch haben Spiele wie jenes zwischen Karpov und Kasparov gezeigt, dass bereits ein Turm und zwei Bauern ausreichen, um gegen zwei kleine Figuren gewinnen zu können.
+
+\begin{equation} \label{eq:piece-values_third-result}
+    L + S > T + B
+\end{equation}
+
+\noindent Aus diesem Grund wird die Gleichung~\ref{eq:piece-values_third-result}, die ferner als Gleichung~\ref{eq:piece-values_third-result-full} dargestellt werden kann, um einen Faktor erweitert, woraus sich Gleichung~\ref{eq:piece-values_third-result-enhanced} ergibt [\cite{Wiki2018}].
+
+\begin{equation} \label{eq:piece-values_third-result-full}
+    T + 2B > L + S > T + B
+\end{equation}
+
+\begin{equation} \label{eq:piece-values_third-result-enhanced}
+    L + S > T + 1.5B
+\end{equation}
+
+\noindent Durch die hier beschriebenen Gleichungen ist der vierte Punkt implizit erfüllt.
+Zu guter Letzt wird noch eine Gleichung benötigt, die die Überlegenheit einer Dame-Bauer-Kombination gegenüber zwei Türmen darstellt.
+Dies ist in Gleichung~\ref{eq:piece-values_final-result-missing} abgebildet.
+
+\begin{equation} \label{eq:piece-values_final-result-missing}
+    Q + B = 2T
```

Comment:
Diese Gleichung stellt nicht die Überlegenheit einer Dame-Bauer Kombination gegenüber 2 Türmen dar. Also entweder Satz oben anpassen oder die Gleichung in D+B > 2T ändern